Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -549,17 +520,17 @@ public override void Subscribe(IEnumerable<Symbol> symbols)
                 return;
             }
 
-            var token = GetAuthenticationToken(JsonConvert.SerializeObject(payload), "GET", "/users/self/verify");
+            var token = GetAuthenticationToken(string.Empty, "GET", "/users/self/verify");
```

Comment:
Nice! 👍 
Should the `GetAuthenticationToken` `body` parameter be remove? Maybe a comment why its an empty string

Reply:
`GetAuthenticationToken` is also used to sign REST API requests and the body is used there.

The empty string for the WebSocket authentication seems to be undocumented 😕 
found the usage in the Coinbase library source:
https://github.com/coinbase/coinbase-pro-trading-toolkit/blob/master/src/exchanges/coinbasePro/CoinbaseProFeed.ts#L235